code/modules/mob/living/simple_animal/hostile/vannatusk.dm

```diff
@@ -65,20 +65,25 @@
 
 /obj/item/bone_dart/vannatusk
 	name = "bone dart"
-	desc = "A sharp piece of bone shapped as small dart."
+	desc = "A sharp piece of bone shaped into a small dart."
 	icon = 'icons/mob/npc/vannatusk.dmi'
 	icon_state = "bonedart"
 
 /mob/living/simple_animal/hostile/vannatusk/attackby(obj/item/O, mob/user)
-	if (stat == DEAD && !crystal_harvested)
-		if(istype(O, /obj/item/surgery/scalpel))
-			visible_message(SPAN_NOTICE("[user] recovers a bluespace crystal from [src]'s remains!"))
-			var/obj/item/bluespace_crystal/C = new(get_turf(src))
-			user.put_in_any_hand_if_possible(C)
-			crystal_harvested = TRUE
+	if(stat != DEAD)
+		return ..()
+	if(istype(O, /obj/item/surgery/scalpel))
+		if(crystal_harvested)
+			to_chat(user, SPAN_WARNING("\The [src]'s crystal has already been harvested!"))
 			return
 
-	..()
+		visible_message(SPAN_NOTICE("[user] recovers a bluespace crystal from [src]'s remains!"))
```

Comment:
this could probably be a visible_message on the user so they can get a personalized messsage